x/gamm/pool-models/balancer/msgs.go

```diff
@@ -10,6 +10,7 @@ import (
 
 const (
 	TypeMsgCreateBalancerPool = "create_balancer_pool"
+	TypeMsgMigrateShares      = "migrate_shares"
```

Comment:
Also a reminder to add this to codec

Reply:
https://github.com/osmosis-labs/osmosis/pull/4051/commits/406dae82d237eee2481d367e925af6e1c038ef0a

Thanks!